other/docker/coverage/run

```diff
@@ -4,8 +4,5 @@ set -eux
 
 read -a ci_env <<<"$(bash <(curl -s https://codecov.io/env))"
 
-BUILD=coverage
-other/docker/sources/build
-docker build -t "toxchat/c-toxcore:$BUILD" -f "other/docker/$BUILD/$BUILD.Dockerfile" .
-
-docker run "${ci_env[@]}" -e CI=true --name toxcore-coverage --rm -t toxchat/c-toxcore:coverage /usr/local/bin/codecov
+. "$(cd "$(dirname "${BASH_SOURCE[0]}")/../sources" && pwd)/run.sh"
```

Comment:
```suggestion
. "$(realpath -- "${BASH_SOURCE[0]}"/../sources/run.sh)"
```

Reply:
Done. This makes it less portable, but I guess it's portable enough.